kpops/components/common/kubernetes_model.py

```diff
@@ -42,6 +42,78 @@ class ImagePullPolicy(str, enum.Enum):
     NEVER = "Never"
 
 
+Weight = Annotated[int, pydantic.Field(ge=1, le=100)]
+
+
+class Operator(str, enum.Enum):
```

Comment:
According to the description in the `SelectorRequirement`:

> [...] If the operator is Gt or Lt, [...]

There should also be a Gt and Lt operator. Am I missing something here? 🤔

Reply:
added